From 7dc1b39994a1a43cf326d37eded61e439cb751a8 Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Thu, 16 Aug 2018 17:48:08 +0200
Subject: [PATCH] TRITON-695 sdcadm tests broken with "Missing assets_* in
 SAPI" Reviewed by: Marsell Kukuljevic <marsell@joyent.com> Approved by:
 Marsell Kukuljevic <marsell@joyent.com>

---
 lib/cli/do_check_config.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/cli/do_check_config.js b/lib/cli/do_check_config.js
index fcf19b7..182c6c5 100644
--- a/lib/cli/do_check_config.js
+++ b/lib/cli/do_check_config.js
@@ -300,7 +300,7 @@ CheckConfig.prototype.execute = function cExecute(opts, args, cb) {
             // value for *_pw.
             services.forEach(function checkSvcCfgVars(s) {
                 if (s === 'cns' || s === 'portolan' || s === 'docker' ||
-                    s === 'cmon' || s === 'volapi') {
+                    s === 'cmon' || s === 'volapi' || s === 'assets') {
                     return;
                 }
                 if (!sdc[s + '_root_pw'] && s !== 'manta' && s !== 'sapi') {
-- 
2.21.0

